cmp/grid/Grid.ts

```diff
@@ -841,4 +870,83 @@ class GridLocalModel extends HoistModel {
             consumeEvent(event);
         }
     };
+
+    //-----------------------------
+    // Support for FullWidthScroll
+    //-----------------------------
+
+    readonly SCROLLBAR_SIZE = 10;
+    readonly scrollerRef = createRef<HTMLDivElement>();
+
+    @observable viewportWidth: number;
+    @observable private isVerticalScrollbarVisible = false;
+
+    private viewportResizeObserver: ResizeObserver;
+
+    @computed
```

Comment:
why is this computed?  don't think we expect this to change,  be observable, or be expensive to check